### PR TITLE
Remove cas.authn.pac4j.saml[0].signLogoutRequests

### PR DESCRIPTION
**Pull request by fmartelli**
signLogoutRequests seems to duplicate signServiceProviderLogoutRequest

### CODE REVIEW

**Comment by leleuj** [comment]
Good catch! Please submit the same PR for master also.

File: api/cas-server-core-api-configuration-model/src/main/resources/META-INF/additional-spring-configuration-metadata.json [hunk 1]
```diff
@@ -149,6 +149,15 @@
         "reason": "management endpoints security under cas.monitor.endpoints.endpoint.*. e.g. cas.monitor.endpoints.endpoint.defaults.access[0].",
         "level": "error"
       }
+    },
+    {
+      "name": "cas.authn.pac4j.saml[0].signLogoutRequests",
```
Comment (mmoayyed) [inline api/cas-server-core-api-configuration-model/src/main/resources/META-INF/additional-spring-configuration-metadata.json]: Should be` single-logout-requests`

File: api/cas-server-core-api-configuration-model/src/main/resources/META-INF/additional-spring-configuration-metadata.json [hunk 2]
```diff
@@ -149,6 +149,15 @@
...
+      "type": "java.lang.Boolean",
+      "description": "Whether logout requests and responses should be signed.",
+      "deprecation": {
+        "reason": "It is a duplication of signServiceProviderLogoutRequest.",
```
Comment (mmoayyed) [inline api/cas-server-core-api-configuration-model/src/main/resources/META-INF/additional-spring-configuration-metadata.json]: Specify the full path to the replacement setting, and please use the same format when naming the setting.

File: api/cas-server-core-api-configuration-model/src/main/resources/META-INF/additional-spring-configuration-metadata.json (hunk 1 above)
Comment (mmoayyed) [reply api/cas-server-core-api-configuration-model/src/main/resources/META-INF/additional-spring-configuration-metadata.json]:
```
    Invalid characters: 'L', 'R'
    Reason: Canonical names should be kebab-case ('-' separated), lowercase alpha-numeric characters and must start with a letter
```

**Comment by mmoayyed** [comment]
I'll cherry-pick this over to master, since the changes are minimal.

**Pull request merged**